.travis.yml

```diff
@@ -114,18 +115,38 @@ jobs:
   fast_finish: true
   include:
     - stage: test
+      name: Ubuntu 16.04 / NodeJS 10 / Python 2
```

Comment:
I think simpler names for all the jobs would be sufficient and better as they will be less likely subject to change (since travis controls the default operating system version and it can be updated in the future). In addition, most of the information added to the build name is already displayed:

<img width="812" alt="Screen Shot 2020-07-25 at 1 02 38 PM" src="https://user-images.githubusercontent.com/40359487/88462172-59213500-ce77-11ea-8fd5-f5e18bd8d585.png">

Something like `(Linux - Python2)` can be sufficient.